src/Aquifer.Jobs/Program.cs

```diff
@@ -21,14 +21,9 @@
 using Polly.Extensions.Http;
 
 var host = new HostBuilder()
-    .ConfigureAppConfiguration((context, builder) => builder
-        .SetBasePath(context.HostingEnvironment.ContentRootPath)
-        .AddJsonFile("appsettings.json", optional: false, reloadOnChange: true)
-        .AddJsonFile(
-            $"appsettings.{context.HostingEnvironment.EnvironmentName}.json",
-            optional: true,
-            reloadOnChange: true)
-    )
+    .ConfigureAppConfiguration((context, builder) => builder.SetBasePath(context.HostingEnvironment.ContentRootPath)
+        .AddJsonFile("appsettings.json", false, true)
```

Comment:
Did your formatter remove the argument names or was that manual?  I'm a fan of explicit argument names for literal types because I've seen too many bugs where `Method(true, false)` should have been `Method(false, true)` and showing the argument names makes this bug much more detectable.

Reply:
It did.